man/man5/zpool-features.5

```diff
@@ -432,5 +432,26 @@ set larger than 128KB, and will return to being \fBenabled\fR once all
 filesystems that have ever had their recordsize larger than 128KB are destroyed.
 .RE
 
+.sp
+.ne 2
+.na
+\fB\fBuserobj_accounting\fR\fR
+.ad
+.RS 4n
+.TS
+l l .
+GUID	org.zfsonlinux:user_object_accounting
```

Comment:
Unless there's a compelling reason otherwise, the string after the `:` should be the same as the user-visible name (i.e. `userobj_accounting` and `user_object_accounting` should be the same string).